packages/utilities/psammead-locales/moment/sr.js

```diff
@@ -10,8 +10,12 @@ moment.updateLocale('sr', {
   relativeTime: {
     past: 'Pre %s',
     m: '1 minuta',
-    mm: 'minuta: %d',
+    mm: '%d minuta',
     h: '1 sata',
-    hh: 'sati: %d',
+    hh: function(number) {
+      return number < 5
+        ? String(number).concat(' sata')
+        : String(number).concat(' sati');
```

Comment:
```suggestion
        : String(number).concat(' ', 'sati');
```

Reply:
What difference would that make?

Reply:
It should be the same, but I think it is safer than adding a space character to the string.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat

Reply:
Updated per your recommendation :)